### DISCUSSION TITLE
Is there a way to supress CHUNKING_SESSION_COOKIE debug logs from output in terminal?

### DISCUSSION THREAD

**Discussion by pnalepa-bishopfox** [nextauthjs/next-auth#9976] [Help]
I'm getting a lot of `CHUNKING_SESSION_COOKIE` logs in the development environment of NextJS app. I'm wondering whether there's a way to hide such logs as they are disturbing the console output a lot and makes debugging more difficult.
```
[19:23:49.080] DEBUG (96800): CHUNKING_SESSION_COOKIE
    metadata: {
      "message": "Session cookie exceeds allowed 4096 bytes.",
      "emptyCookieSize": 163,
      "valueSize": 4307,
      "chunks": [
        4096,
        537
      ]
    }
```
I would like to get rid of it. Maybe I should not do that?

**Comment by alasdairmackenzie** [comment]
I had the same question.

Just looked at the code that logs it out and realised you can set debug to `false` or override the logger behaviour in the auth config - `auth.config.ts` for me. It would be nice to have the option to suppress individual log messages.

```js
logger: {
  debug: (msg) => {
    if (msg.includes("CHUNKING_SESSION_COOKIE")) {
      return;
    }
    console.debug(msg);
  },
},
debug: false,
```

See the debugging docs for details
https://authjs.dev/guides/debugging
